### DISCUSSION TITLE
link tugas capstone tidak bisa d akses

### DISCUSSION THREAD

**Discussion by SitiNurbaya28** [teamalgoritma/community#718] [Q&A] [answered]
link tugas capstone saya tidak bisa d akses,,udah coba buka logs nya d shiny apps tapi masih bingung cara hendle error nya,,,
 di mohon bantuan nya,,terima kasih,,,

[capstone_shiny-logs.txt](https://github.com/teamalgoritma/community/files/11998535/capstone_shiny-logs.txt)

**Comment by SitiNurbaya28** [comment]
![Screenshot 2023-07-10 133305](https://github.com/teamalgoritma/community/assets/133105934/1a87e7f6-48b6-4c2a-be1e-697a50a5a204)

**Comment by arkanayudhistira** [comment]
Dear @SitiNurbaya28,

Berdasarkan logs yang disampaikan, sepertinya terdapat error yang tertulis `object 'retail' not found`, apakah variabel `retail` telah didefinisikan pada file `global.R`? karena kemungkinan Shiny tidak menemukan variabel tersebut.

**Reply by SitiNurbaya28** [reply]
object 'retail' sudah saya definisikan di global.R mas,,,apa ada yang salah d penulisan apa gmna ya mas,,


![Screenshot 2023-07-11 121707](https://github.com/teamalgoritma/community/assets/133105934/8a118ed8-d1df-4868-9fb4-f1a77fff9b13)

**Reply by arkanayudhistira** [reply, marked as answer]
Dear @SitiNurbaya28 

Silahkan untuk mengupload file pengerjaan dashboardnya sehingga dapat kami periksa lebih lanjut untuk masalahnya, terima kasih.

**Reply by kevbow** [reply]
Halo @SitiNurbaya28, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
